packages/dm-core/src/types.ts

```diff
@@ -98,6 +98,7 @@ export type TUserIdMapping = { userId: string; username: string }
 export type TViewConfig = {
   type: string
   scope: string
+  label?: string
```

Comment:
Not sure if we should add a label attribute to ViewConfig, my impression is that these things are plugin-specific. 

So maybe extends the TViewConfig or create a new type that includes the view config? Then this will be plugin specific for attribute-selector that can also include a label. 

Extends:
```
"type": "AttributeSelectorItem",
"label":"",
 ...viewConfig...
```

Compose:
```
"type": "AttributeSelectorItem",
"label": "",
"viewConfig": {
  ...
}
```

Reply:
fixed